modules/sdk-coin-sui/src/lib/txnSchema.ts

```diff
@@ -0,0 +1,27 @@
+import joi from 'joi';
+
+const RequestAddDelegationArgumentsSchema = joi.array().length(4).items(joi.any()).required();
+
+const SuiObjectRefSchema = joi.object({
+  objectId: joi.string().required(),
+  version: joi.number().required(),
+  digest: joi.string().required(),
+});
+
+export const RequestAddDelegationTransactionSchema = joi.object({
+  package: joi.alternatives(SuiObjectRefSchema).match('one').required(),
+  module: joi.string().required(),
+  function: joi.string().required(),
+  typeArguments: joi.array().items(joi.any()).optional(),
```

Comment:
Can't we use a stricter type than any here?

Reply:
yes, can be done 👍